testsuites/syncgateway/functional/tests/test_crud.py

```diff
@@ -21,7 +22,10 @@
 @pytest.mark.session
 @pytest.mark.parametrize('sg_conf_name, deletion_type', [
     ('sync_gateway_default_functional_tests', 'tombstone'),
-    ('sync_gateway_default_functional_tests', 'purge')
+    ('sync_gateway_default_functional_tests', 'purge'),
+    ('sync_gateway_default_functional_tests_no_port', 'tombstone'),
+    ('sync_gateway_default_functional_tests_no_port', 'purge'),
+    ('sync_gateway_default_functional_tests_couchbase_port', 'purge')
```

Comment:
I'm unclear which one of these implies ssl testing - no_port?  couchbase_port?  From the names it's hard to interpret what the intended test is - can we come up with better names for no_port and couchbase_port?

Reply:
What  I implied is 
sync_gateway_default_functional_tests_no_port . -> config without port -> Test runs only if ssl enabled.Protocol get replaced with 'couchbases'  if ssl enabled, otherwise it uses http. This is generic config file with no port , so I think we can use this name. 

sync_gateway_default_functional_tests_couchbase_port -> I know this is little confusing, I tried to have short name. This config uses hard code protocol 'couchbase' and uses port . Port variable varies based on condition ssl enabled/ssl disabled. To make it short I used couchbase_port instead of couchbase_protocol_with_port. 
I renamed to 'sync_gateway_default_functional_tests_couchbase_protocol_with_port'